src/CircFit.jl

```diff
@@ -1,7 +1,36 @@
 module CircFit
 import LsqFit: lmfit
+import Statistics: var, cov
 
-export circfit
+export circfit, Kasa
+
+@enum Method begin
+    Kasa
+end
+
+"""
+Using analytical solution of Kasa
+https://ieeexplore.ieee.org/document/1246564
+x y values are arrays
+"""
+function circfit(::Type{Val{Kasa}},x::AbstractArray,y::AbstractArray)
+    n = length(x)
+    x² = x.^2
+    y² = y.^2
+    
+    A = n*(n-1)*var(x) 
+    B = n*(n-1)*cov(x,y) 
+    C = n*(n-1)*var(y) 
+    D = n*(n-1)/2*(cov(x,y²)+cov(x,x²)) 
+    E = n*(n-1)/2*(cov(y,x²)+cov(y,y²)) 
+
+    ACB2 = (A*C-B^2)
+    am = (D*C - B*E) / ACB2
+    bm = (A*E - B*D) / ACB2
+    rk = sqrt(sum(((x.-am).^2 + (y.-bm).^2)/n))
```

Comment:
This allocates 4 temporary arrays (vs. 1 if you used `.+` and `./`) and is technically susceptible to spurious overflow (e.g. if `x - am` is `1e200` it will give `Inf`, although that would probably also overflow in your earlier calculations).  Would probably be better to use:
```jl
rk = hypot(stdm(x, am, corrected=false), stdm(y, bm, corrected=false))
```
which I think allocates no temporaries.